page-objects/pages/CutePage.js

```diff
@@ -1,47 +1,46 @@
-class cute{
-    get subscribeLabel(){
-        return $('.c-subscribe-email__input')
-    }
-
-    get emailButton(){
-        return $('.c-subscribe-email__submit')
-    }
-
-    get trendingArticle1(){
-        return $$('.c-recommendation__link')[0]
-    }
-
-    get trendingArticle2(){
-        return $$('.c-recommendation__link')[1]
-    }
-
-    get moreArticle1(){
-        return $$('.c-more-from-post')[0]
-    }
-
-    get moreArticle2(){
-        return $$('.c-more-from-post')[1]
-    }
-
-    get moreArticle3(){
-        return $$('.c-more-from-post')[2]
-    }
-
-    get facebookButton(){
-        return $('.qa-facebook__icon')
-    }
-
-    get twiterButton(){
-        return $('.qa-twitter__icon')
-    }
-
-    get mailButton(){
-        return $('.qa-mail__icon')
-    }
-
-    get addusEmailSent() {
-        return $('.hcard-download')
-    }
-
+class cute {
+  get subscribeLabel() {
```

Comment:
I think a more suggestive name for this variable should be subscribeEmailInput. The selector suggests being input and the name suggests to be a label. It is confusing